src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTHttpClientHandler.cs

```diff
@@ -0,0 +1,475 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
You know, given that this decision involves some discussion, I'm fine with the file names you have now, and we can visit for (possible) file renames later, after this is merged.